.eslintrc.json

```diff
@@ -0,0 +1,32 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true,
+        "node": true
+    },
+    "extends": [
+        "plugin:react/recommended",
+        "airbnb"
```

Comment:
I think we should consider adding typescript-react here as well.

Reply:
Absolutely, I can't find the link to that package on npm. Could you provide the link?